qiskit_ibm_runtime/transpiler/passes/scheduling/__init__.py

```diff
@@ -332,16 +360,9 @@
 there are fast-path blocks that will be performed in parallel they currently *will not*
 be padded out by the scheduler to ensure they are of the same duration in Qiskit
 
-.. jupyter-execute::
-
-    dd_sequence = [XGate(), XGate()]
-
-    pm = PassManager(
-        [
-            ALAPScheduleAnalysis(durations),
-            PadDynamicalDecoupling(durations, dd_sequence),
-        ]
-    )
```

Comment:
I divided the code into two blocks to display both circuits. Before, even though we ran `.draw()` twice, we were only showing the last circuit as we can see at the end of this page: https://docs.quantum.ibm.com/api/qiskit-ibm-runtime/qiskit_ibm_runtime.transpiler.passes.scheduling